src/main/java/nextstep/member/ui/config/AuthArgumentResolver.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.member.ui.config;
+
+import nextstep.member.application.JwtTokenProvider;
+import org.springframework.core.MethodParameter;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+@Component
+public class AuthArgumentResolver implements HandlerMethodArgumentResolver {
+
+    private final JwtTokenProvider jwtTokenProvider;
+
+    public AuthArgumentResolver(JwtTokenProvider jwtTokenProvider) {
+        this.jwtTokenProvider = jwtTokenProvider;
+    }
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.hasParameterAnnotation(Auth.class);
+    }
+
+    @Override
+    public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) throws Exception {
+        String authorizationHeader = webRequest.getHeader("authorization");
```

Comment:
```suggestion
        String authorizationHeader = webRequest.getHeader(HttpHeaders.AUTHORIZATION);
```
문자열 대신 스프링이 제공하는 enum을 쓰는 방법도 있겠네요.